.circleci/config.yml

```diff
@@ -86,6 +106,30 @@ jobs:
             - ~/.cache/yarn
       - run: yarn build:webpack
 
+  build-e2e:
```

Comment:
This may supersede the other `build` job later. When that happens, maybe add the `--env.entry` flags later for the specific files that changed.